### ISSUE TITLE
About "xxHash" not found

### ISSUE THREAD

**Issue by CN-HYC** [osu-crypto/FuzzyPSI#1]
Could not find a package configuration file provided by "xxHash"

**Comment by yangchunmian123** [comment]
I have the same problem, have you solved it please?

**Comment by xade93** [comment]
Yeah apparently `xxHash` is a prerequisite which author did not mention at all. 
You can install `xxHash` from `vcpkg` according to its official documentation. After that you can add `vcpkg`'s cmake toolchain manually into FuzzyPSI's cmakelist via something like:
```cmake
set(CMAKE_TOOLCHAIN_FILE "/work/vcpkg/scripts/buildsystems/vcpkg.cmake"
  CACHE STRING "Vcpkg toolchain file")
```
